angular-client/src/components/current-display/current-display.component.html

```diff
@@ -0,0 +1,9 @@
+<info-background title="Current" icon="Bolt">
+  <hstack style="width: 40%">
+    <div class="battery-display-body">
+      <typography variant="info-value" [content]="amps.toString()" />
```

Comment:
also for this and the latency, I think we could bump a little space between the two elements to make it look a bit cleaner